### PR TITLE
modify all structs/functions in this file to be unexported

### PR DESCRIPTION
**Pull request by zouyee**

### CODE REVIEW

File: xds/internal/client/client_watchers_endpoints.go [hunk 1]
```diff
@@ -91,3 +96,66 @@ func (c *Client) WatchEndpoints(clusterName string, cb func(EndpointsUpdate, err
...
 	return c.watch(wi)
 }
+
+// BuildEndpointsUpdate builds EndpointsUpdate.
+func BuildEndpointsUpdate(dropPercents []uint32) *EndpointsUpdate {
+	ret := &EndpointsUpdate{}
+
+	for i, d := range dropPercents {
+		drop := OverloadDropConfig{
+			Category:    fmt.Sprintf("test-drop-%d", i),
```
Comment (easwars) [inline xds/internal/client/client_watchers_endpoints.go:106]: These are used only in tests, right? These should be in some testutil file.

Comment (zouyee) [reply xds/internal/client/client_watchers_endpoints.go:106]: got it

**Comment by easwars** [comment]
Could you make travis happy please.

**Comment by dfawley** [comment]
I kicked travis and it seems to be happy now.

**Pull request closed without merging**